keyboards/mmkeyboard/class60/soldered/info.json

```diff
@@ -0,0 +1,178 @@
+{
+    "manufacturer": "MMkeyboard",
+    "keyboard_name": "Class60",
+    "maintainer": "lexbrugman",
+    "bootloader": "uf2boot",
+    "build": {
+        "lto": true
+    },
+    "development_board": "bluepill",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": true,
+        "bootmagic": true,
+        "extrakey": true,
+        "haptic": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "B9",
+        "num_lock": "B8",
+        "on_state": 0,
+        "scroll_lock": "B11"
+    },
+    "matrix_pins": {
+        "cols": ["B1", "B0", "A7", "A1", "A2", "A3", "A4", "A5", "B12", "B13", "A10", "A9", "A8", "B14", "B15"],
+        "rows": ["B6", "B7", "C13", "B10", "C14", "B5"]
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "url": "https://www.mmkeyboard.com/products/mm-class60-retro-custom-mechanical-keyboard",
+    "usb": {
+        "device_version": "0.0.1",
+        "force_nkro": true,
+        "pid": "0x3876",
+        "vid": "0x3874"
```

Comment:
This is close to being assigned.  The further past 0x4000, the better.

Reply:
These ids are taken from the vial firmware for this same board, I thought it makes sense to keep them the same.

Reply:
vial plays a lot faster and looser with these.   And by close right now, it looks like USB-IF is assigned to about 0x373x, so this will be assigned soon.   I don't feel comfortable using this vendor ID.

Reply:
Check, done!